frontend/src/i18n/en.json

```diff
@@ -173,6 +173,7 @@
     "globalRules": "This is a global set of allow and disallow rules. They apply to every user. You can define specific rules on each user's settings to override this ones.",
     "allowSignup": "Allow users to signup",
     "createUserDir": "Auto create user home dir while adding new user",
+    "singleClick": "Use single clicks to open folders",
```

Comment:
```suggestion
    "singleClick": "Use single clicks to open files and directories",
```

Reply:
I will check in the evening, but I think those changes does not affect files, only directories.
I can rename `folders` to `directories` anyway. 🙂